myst_nb/transform.py

```diff
@@ -195,3 +197,19 @@ def cell_output_to_nodes_inline(
                 to_add.append(JupyterWidgetViewNode(view_spec=data))
 
     return to_add
+
+
+def cell_output_to_nodes(outputs, data_priority, write_stderr, dir, thebe_config):
+    from jupyter_sphinx.ast import cell_output_to_nodes
```

Comment:
short docstring so that we know this is basically just mimicking `jupyter-sphinx` w/ some extra logic

Reply:
add documentation, add type annotations

Reply:
also we should consider if/how this can be upstreamed